src/app/components/harvest/screens/metadata-review/metadata-review.component.html

```diff
@@ -75,33 +75,51 @@ <h3>Review</h3>
 </div>
 
 <div class="mt-3 d-flex justify-content-between">
-  <button
+  <div>
+    <button
+      id="harvest-abort-button"
+      class="btn btn-outline-danger me-2"
+      [disabled]="transitioningStage"
+      (click)="abortUpload(abortUploadModal)"
+    >
+      Abort
+    </button>
+
+    <button
+    id="harvest-upload-more-files-button"
     class="btn btn-outline-primary"
     [disabled]="transitioningStage"
     (click)="upload(uploadModal)"
-  >
-    Upload more files
-  </button>
-  <button
-    *ngIf="harvest.isMappingsDirty"
-    class="btn btn-primary"
-    [disabled]="transitioningStage"
-    (click)="extraction(extractionModal)"
-  >
-    Check changes
-  </button>
+    >
+      Upload more files
+    </button>
+  </div>
+
   <!-- TODO Modal popup when hasUnsavedChanges -->
-  <button
-    class="btn"
-    [class.btn-primary]="!harvest.isMappingsDirty"
-    [class.btn-warning]="harvest.isMappingsDirty"
-    [disabled]="transitioningStage"
-    (click)="processing(processingModal)"
-  >
-    {{
-      harvest.isMappingsDirty ? "Continue without checking changes" : "Continue"
-    }}
-  </button>
+  <div>
+    <button
+      id="harvest-check-changes-button"
```

Comment:
why was this id added?

Reply:
This id is not necessary. I have since reverted this change